### PR TITLE
Mds cert dates

### PR DESCRIPTION
**Pull request by ohlookadollar**
Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: scst-store/additional.hbs.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ This section includes the following topics:
 - [Ingress support](ingress.hbs.md)
 - [Custom certificate configuration](custom-cert.hbs.md)
 - [TLS configuration](tls-configuration.hbs.md)
+- [Cert rotation](cert-rotation.hbs.md)
```
Comment (Manifaust) [inline scst-store/additional.hbs.md]: Adding it to `additional.hbs.md` is good.

Also you need to add it to `toc.md`. Showing up in the table of contents is what the docs teams wants for every doc.

**Review by Manifaust — APPROVED** [review]

File: scst-store/additional.hbs.md (hunk 1 above)
Comment (xtreme-debbie-chen) [inline scst-store/additional.hbs.md]: `Cert` -> `Certificate`

File: scst-store/cert-rotation.hbs.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Certificate Rotation
+
+This topic describes TLS Certificate Rotation for Supply Chain Security Tools (SCST) - Store.
+
+## Certificates
+
+By default the `use_cert_manager` setting is set to `"true"`.
```
Comment (xtreme-debbie-chen) [inline scst-store/cert-rotation.hbs.md]: `By default the` -> `By default, the`

File: scst-store/cert-rotation.hbs.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+metadata_store:
+  ca_cert_duration: 8760h
+  ca_cert_renew_before: 1h
+  api_cert_duration: 2160h
+  api_cert_renew_before: 24h
+```
+
+Where `ca_cert_duration` is the duration that the ca certificate will be valid for. Must be given in h, m, or s. Default value is 8760h
+
+Where `ca_cert_renew_before` is the time when to renew the ca certificate before the expiration time. Must be given in h, m, or s. Default value is 1h
```
Comment (xtreme-debbie-chen) [inline scst-store/cert-rotation.hbs.md]: Can you clean up this part (`the time when to renew`)? It's not quite making sense to me.

File: scst-store/cert-rotation.hbs.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+Where `api_cert_duration` is the duration that the api certificate will be valid for. Must be given in h, m, or s. Default value is 2160h
+
+Where `api_cert_renew_before` is the time when to renew the api certificate before the expiration time. Must be given in h, m, or s. Default value is 24h
+
+Note: The above settings only take effect when `use_cert_manager` is `"true"`. If the `use_cert_manager` is not set, it will default to `"true"`
```
Comment (xtreme-debbie-chen) [inline scst-store/cert-rotation.hbs.md]: Add a period at the end of this sentence.

**Review by xtreme-debbie-chen — APPROVED** [review]

**Pull request merged**